core/templates/dev/head/components/top_navigation_bar/TopNavigationBarDirective.js

```diff
@@ -60,7 +60,7 @@ oppia.directive('topNavigationBar', [function() {
             $window.location = GLOBALS.loginUrl;
           }, 150);
         };
-
+        $scope.isSidebarShown = true;
```

Comment:
Change to $scope.sidebarIsShown to indicate it's not a function (since it doesn't start with a verb).